angular-workspace/projects/example-client-app/src/app/customapp/customapp.component.html

```diff
@@ -202,9 +202,6 @@
                     <nimble-icon-check title="String 1"></nimble-icon-check>
                 </nimble-table-column-text>
                 <nimble-table-column-anchor
-                    nimbleRouterLink
-                    [queryParams]="{ word: 'foo' }"
```

Comment:
There's a text field below whose purpose was to display the query params so I think it can be deleted now.

Reply:
Oops, good catch.